sdcm/cluster.py

```diff
@@ -3363,6 +3363,15 @@ def get_all_tables_with_cdc(self, db_node: BaseNode) -> List[str]:
 
         return ks_tables_with_cdc
 
+    def run_node_benchmarks(self):
+        if not self.params.get("run_node_benchmarks"):
+            return
+
+        self.node_benchmark_manager.add_nodes(self.nodes)
+        self.node_benchmark_manager.install_benchmark_tools()
+        self.node_benchmark_manager.run_benchmarks()
+        self.node_benchmark_manager.print_benchmark_results_to_log()
+
```

Comment:
Please, place original method in the same place to reduce number of changed code lines.

Reply:
Fixed with rebase